playbooks/openshift-glusterfs/private/setup_nodes.yml

```diff
@@ -10,6 +10,12 @@
   - import_role:
       name: openshift_node
       tasks_from: glusterfs.yml
+  - import_role:
+      name: openshift_node
+      tasks_from: storage_plugins/iscsi.yml
```

Comment:
`ERROR! Could not find specified file in role: tasks/iscsi.yml`

Reply:
Done